Dockerfile

```diff
@@ -82,30 +51,21 @@ FROM temporal-server AS temporal-auto-setup
 CMD /start.sh autosetup
 
 # Temporal CLI
-FROM alpine AS temporal-tctl
+FROM temporalio/base-server AS temporal-tctl
 
-COPY --from=builder /temporal/tctl /usr/local/bin
+COPY --from=temporal-builder /temporal/tctl /usr/local/bin
 
 ENTRYPOINT ["tctl"]
 
 # All temporal tool binaries
-FROM alpine AS temporal-admin-tools
```

Comment:
This line still needs to be there.

Reply:
base-admin-tools is now used, that is from alpine. Don't see alpine used anywhere else.

Reply:
it would need to be `FROM temporalio/base-admin-tools as temporal-admin-tools` right? I think there still needs to be FROM something here

Reply:
probably also be best to include a version in the base-admin-tools image too

Reply:
This is what I meant. And don't forget to add version.

Reply:
fixed